catalog/admin/module/catalog-admin-module-sources/src/main/webapp/js/model/Source.js

```diff
@@ -35,6 +35,8 @@ define([
             idAttribute: 'name',
             initialize: function () {
                 this.set('currentConfiguration', undefined);
+                this.set('currentUrl', undefined);
```

Comment:
Worst case scenario you could return the two new attributes as defaults, leave the others as sets on initialize.